src/Worker.js

```diff
@@ -89,6 +96,11 @@ function setup(tr, babel) {
   }
 
   const module = require(tr);
+
+  if (babelRegister) {
+    babelRegister.revert();
+  }
```

Comment:
This test: 

https://github.com/facebook/jscodeshift/blob/57a9d9c731429da624655472290a7b4802e7e8d7/bin/__tests__/jscodeshift-test.js#L194-L213

Catches that while reverting the babel register here works for any require/import that would be run at `require` time, it wouldn't catch any that would only be executed at run time.

This part isn't too significant, so supporting that case probably more important than worrying about reverting the register? Happy to 🔪